serum-390/soen390-team16-godzilla
# Authentication backend and log in screen

# Description

Adds basic http authentication to the app. Locks all APIs and client behind login screen.

## Link to work item

#67

## How to Test

1. Run the app
2. Test any route with no authentication => should be returned a 401 and/or sent to default Spring login screen.
3. Test routes with authentication, should be allowed through the security filter, use credentials:
    - USERNAME: `demo`
    - PASSWORD: `demo`
